packages/ckeditor5-link/src/linkediting.ts

```diff
@@ -90,7 +90,7 @@ export default class LinkEditing extends Plugin {
 		editor.config.define( 'link', {
 			allowCreatingEmptyLinks: false,
 			addTargetToExternalLinks: false,
-			toolbar: [ 'linkPreview', '|', 'editLink', 'unlink' ]
+			toolbar: [ 'linkPreview', '|', 'editLink', 'linkProperties', 'unlink' ]
```

Comment:
There is no point in adding this button if the integrator does not specify manual decorators.

Reply:
I'd filter it out later near:

```ts
toolbarView.fillFromConfig( editor.config.get( 'link.toolbar' )!, editor.ui.componentFactory );
```